src/models/task/postgres.js

```diff
@@ -8,15 +8,15 @@ const TaskRepo = (postgres) => {
             duration int NOT NULL,
             due_date timestamptz NOT NULL,
             completed boolean,
-            completed_time timestamp, 
+            completed_time timestamptz, 
             scheduled boolean,
-            scheduled_time timestamp,
+            scheduled_time timestamptz,
             calendar_id text,
             event_id text,
-            start_time timestamp,
-            end_time timestamp,
-            created_time timestamp DEFAULT NOW(),
-            updated_time timestamp DEFAULT NOW()
+            start_time timestamptz,
+            end_time timestamptz,
+            created_time timestamptz DEFAULT NOW(),
+            updated_time timestamptz DEFAULT NOW()
```

Comment:
Not saying this is wrong, but what is the reason behind all this? Do we need to store timezone info if our contract is that they will be stored in UTC? Or is this needed for Moment?

Reply:
Currently, all my Moment math is done in the user's local time zone because that made it easier for me to manually compare the POST request result to my actual calendar and debug the algorithm. I'll convert everything to be done in UTC before I merge into `master`.